help/zulip-cloud-billing.md

```diff
@@ -111,6 +130,14 @@ eligibility.
 
 1. Fill out the requested information, and click **Submit**.
 
+!!! tip ""
+
+     If you don't see the **Request sponsorship** link in the gear menu,
+     update your [organization type](/help/organization-type) to one of
+     the eligible types of organizations [listed
+     above](#free-and-discounted-zulip-cloud-standard) that best fits
+     your organization.
```

Comment:
I moved this to be a tip and not one of the instructions. And did the same for the request education pricing instructions tab.

Reply:
Works for me! I feel like we could somehow use the word "organization" fewer times, but I'm not immediately able to come up with a better variant.